### ISSUE TITLE
Include requirements files

### ISSUE THREAD

**Issue by AnnyBon** [kdave/audio-compare#1]
Can you please include requirements.in and requirements.txt files?

**Comment by kdave** [comment]
Yeah, though i"m not sure how much useful it is for a simple script. The dependencies for requirements.txt are correlation and numpby, I can't find what should be in requirements.in.

**Comment by leoroverpi** [comment]
Sorry ,I'm a very new to write code and python
I try to use this code to verify my experiment
But I'm not sure how to use this
Can you explain how to use from the start please?
thx
